### PR TITLE
Python packages updates

### PR DESCRIPTION
**Pull request by SelinaTII**
Updated the following python packages in modules/utils/package/:
* certifi --> v 2022.5.18.1 to 2022.12.7
* Pillow --> v 9.1.1 to 9.3.0
* cryptography --> v 39.0.0 to 39.0.1
* xgboost --> v 1.6.1 to 1.7.0
* scipy --> v 1.8.1 to 1.10.1
* torch --> v 1.11.0 to 1.13.1
* joblib --> v 1.1.0 to 1.2.0
* setuptools --> v 62.6.0 to 65.5.1

modules/sc-mesh-secure-deployment/src/1_5/main.py is running fine for MS 1.5

### CODE REVIEW

File: modules/utils/package/README.md [hunk 1]
```diff
@@ -1,38 +1,39 @@
...
+|torch| 1.13.1    |    machine_learning_packages|
+|dpkt| 1.9.7.2   |    machine_learning_packages|
+|cycler| 0.11.0    |    machine_learning_packages|
+|fonttools| 4.33.3    |    machine_learning_packages|
+|joblib| 1.2.0     | machine_learning_packages |
+|kiwisolver| 1.4.3     | machine_learning_packages |
+|matplotlib| 3.5.2     | machine_learning_packages |
+|packaging| 21.3      | machine_learning_packages |
+|pyparsing| 3.0.9     | machine_learning_packages |
+|sklearn| 0         | machine_learning_packages |
```
Comment (joenpera) [inline modules/utils/package/README.md:35]: sklearn is deprecated package and scikit-learn should be used instead

Comment (martin-tii) [reply modules/utils/package/README.md:35]: @Willtl please advise, and take the necessary actions

**Review by Willtl — COMMENTED** [review]
@joenpera, you can proceed with scikit-learn as you mentioned previously.

**Review by joenpera — APPROVED** [review]
Let's see when these are really imported as python libs.

**Review by martin-tii — APPROVED** [review]

**Pull request merged**